voripos-provision.sh

```diff
@@ -6,6 +6,7 @@ set +x
 
 VORIPOS_PROVISION_VERSION=0.10.0
 VORI_API_ROOT="${VORI_API_ROOT:-https://api.vori.com/v1}"
+DOMAIN_FILE_PATH="~/Library/Containers/com.vori.VoriPOS/Data/Library/Application Support/domain"
```

Comment:
Use $HOME instead of tilde for maximum compatibility. Also, capitalize Domain (even though it doesn’t matter on macOS).